### DISCUSSION TITLE
Question about test set

### DISCUSSION THREAD

**Discussion by namtranlab** [deepmodeling/deepmd-kit#1833] [Q&A]
Hi Everyone,

I checked the forum that #217 If I have a system with a number of sets in the _training_data_, the last set is used as testing data. Does it mean that the last set will not be used during the training? 

If this is the case, should we have the data of RSME for this test set in the output? I only found the data for the validation set in the output. I see in some publications plotting the evolution of RSME on the test set during the training. It is the data for the validation set in the output, isn't it?

Also, It seems that the command _dp test_ only picks the data of the last set in a system? I tried to change the value of _-n NUMBER_ but it seems not working.

Best regards

**Comment by njzjz** [comment]
This behavior has been deprecated since v2.0.0.
